lib/ws.js

```diff
@@ -21,6 +21,9 @@ class BinanceWS {
             ticker: symbol => `${symbol.toLowerCase()}@ticker`,
             allTickers: () => '!ticker@arr'
         };
+
+        // Reference to the setInterval timer for sending keep alive requests in onUserData
+        this.userDataRefreshIntervalId;
```

Comment:
Do you expect this to be accessed as a public property?  If not, you should probably prefix it with an underscore.  Also, seems weird to just have this on it's own without assigning anything to it as it's already `undefined`.  Perhaps initialize it to `null` or `false`?  Or perhaps change it to an object, rename it, and make the interval a property?  See my other comment in `onUserData`

Reply:
Should be clearer now, let me know your thoughts.